.github/workflows/on-push-or-pull.yml

```diff
@@ -48,24 +48,18 @@ jobs:
         runs-on: ubuntu-latest
         needs: one_run
         steps:
-            - uses: actions/checkout@v2
-            - uses: actions/cache@v2
-              id: cache
+            - uses: actions/checkout@v3
+            - name: Yarn setup
+              uses: DerYeger/yarn-setup-action@v2.0.0
```

Comment:
just curious, what's this and why do we need it?

Reply:
This is for installing and caching yarn deps